teachertool/src/components/ProjectWorkspace.tsx

```diff
@@ -7,18 +7,41 @@ import { Toolbar } from "./Toolbar";
 import { ShareLinkInput } from "./ShareLinkInput";
 import { MakeCodeFrame } from "./MakecodeFrame";
 import { CatalogOverlay } from "./CatalogOverlay";
+import { Strings, Ticks } from "../constants";
+import { setRunOnLoad } from "../transforms/setRunOnLoad";
 
 const ProjectName: React.FC = () => {
     const { state: teacherTool } = useContext(AppStateContext);
     const projectName = getSafeProjectName(teacherTool);
     return projectName ? <div className={css.projectName}>{projectName}</div> : null;
 };
 
+const ProjectToolbar: React.FC = () => {
+    const { state: teacherTool } = useContext(AppStateContext);
+    const { runOnLoad } = teacherTool;
+
+    const onRunOnLoadToggled = (checked: boolean) => {
+        pxt.tickEvent(Ticks.RunOnLoad, { checked: checked ? "true" : "false" });
```

Comment:
```suggestion
        pxt.tickEvent(Ticks.RunOnLoad, { newValue: checked ? "true" : "false" });
```

Just for clarity. When I see "checked" my immediate next thought is "is that the before checked or the after checked"